mt-st

```diff
@@ -0,0 +1,38 @@
+#mt bash completion by Paweł Marciniak <sunwire<at>gmail.com>
+#
+
+_mt () {
+    local cur prev words cword
+    _init_completion || return
+
+    #possible commands
+    commands="weof wset eof fsf fsfm bsf bsfm fsr bsr fss bss rewind offline rewoffl eject retension eod seod seek tell status erase setblk lock unlock load compression setdensity drvbuffer stwrthreshold stoptions stsetoptions stclearoptions defblksize defdensity defdrvbuffer defcompression stsetcln sttimeout stlongtimeout densities setpartition mkpartition partseek asf stshowopt"
+
+    if [ -x "$(command -v udevadm)" ]; then
```

Comment:
Does this mean that only when udevadm is installed, we can complete tape devices? Or do I misunderstand here?

Reply:
Yes. udevadm is run only when is installed,